packages/scaffold-utils/package.json

```diff
@@ -33,15 +33,17 @@
     }
   },
   "scripts": {
-    "build:clean": "rm -rf dist",
-    "build:scaffold-utils": "tsc --build",
+    "clean": "rm -rf dist",
+    "build": "tsc --build",
     "watch": "tsc --watch",
     "typecheck": "tsc --noEmit",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
-    "@web3modal/core": "5.0.7",
-    "@web3modal/polyfills": "5.0.7",
+    "@coinbase/wallet-sdk": "4.0.3",
```

Comment:
why add this dependency?

Reply:
https://github.com/WalletConnect/web3modal/blob/4a242fa2da1fef410e8df7c489204b0f3e4ad68c/packages/scaffold-utils/src/EthersTypesUtil.ts#L2